upload/admin/controller/extension/shipping/zasilkovna.php

```diff
@@ -1365,4 +1367,136 @@ private function removeInvalidKeyFromPostData()
 		return $postCopy;
 	}
 
+	/**
+	 * @throws ReflectionException
+	 */
+	public function add_vendor()
+	{
+		$data = $this->initPageData(self::ACTION_ADD_VENDOR, 'Nový vendor');
+		$countryCode = $this->request->get[self::PARAM_COUNTRY];
+
+		$data['vendors'] = $this->getVendorsByCountry($countryCode);
+		$data['internal_vendors'] = $this->getInternalVendorsByCountry($countryCode);
+		$data['action'] = self::ACTION_ADD_VENDOR;
+
+		//mock data
+		$vendorId = 106;
+		$data['weight_rules'] = $this->getVendorWeightRules($vendorId);
```

Comment:
Proč načítáš weight_rules ? pro založení nového vendora to nepotřebuješ.  Jak jsme se dohodli, v rámci tohoto PR neřešíme editaci, jen založení nového vendora.